specs/spec_windows.go

```diff
@@ -1,5 +1,13 @@
 package specs
 
+// Temporary Windows version of the spec in lieu of opencontainers/specs having
+// Windows support currently.
+
+type (
+	PlatformSpec WindowsSpec
+	ProcessSpec  Process
```

Comment:
Ah, no it's not missing as it's defined in this file itself on Windows (as opencontainer/specs isn't in a usable state on Windows).